ga4gh/datamodel/variants.py

```diff
@@ -450,9 +450,10 @@ def convertVariant(self, record):
         if record.id is not None:
             variant.names = record.id.split(';')
         variant.start = record.start          # 0-based inclusive
-        variant.end = record.stop             # 0-based exclusive
+        variant.end = record.start + len(record.ref)
```

Comment:
Indeed! Great to have an expert eye on the VCF code, thanks for pointing this out.